interp/interp_test.go

```diff
@@ -906,6 +906,18 @@ var runTests = []runTest{
 		`mkdir a; ln -s a b; [[ $(cd a && pwd) == "$(cd b && pwd)" ]]; echo $?`,
 		"1\n",
 	},
+	{
+		`pwd -a`,
+		"invalid option: \"-a\"\nexit status 2",
+	},
+	{
+		`pwd -L -P`,
+		"pwd cannot take multiple arguments\nexit status 1",
+	},
+	{
+		`mkdir a; ln -s a b; [[ "$(cd a && pwd -P)" == "$(cd b && pwd -P)" ]]`,
```

Comment:
done!

Reply:
maybe I wasn't clear - I meant literally duplicating this test case, but instead of using `pwd -P`, using `pwd` and `pwd -L`. They should presumably say that the directories are different.

Reply:
ah I see, my mistake — should be fixed now